common/planet_classes/hfe_pc_pirate.txt

```diff
@@ -13,10 +13,16 @@ hfe_pc_pirate_base = {
 		# district_city_max = -1000
 	}
 	spawn_odds = 0
-	can_be_moon = no
 	chance_of_ring = 0.0
-	#city_color_lut = "gfx/portraits/misc/colorcorrection_neutral.dds"
+	city_color_lut = "gfx/portraits/misc/colorcorrection_nuked.dds"
 	planet_size = 5
 	colonizable = yes
 	starting_planet = no
+	is_artificial_planet = yes
```

Comment:
what this mean?

Reply:
It's similar to Habitable Structures, in that the game will differentiate it from a natural planet. This excludes it from things like Abandoned Terraforming Equipment and things of that nature.